swagger-spec/nodes/providers_list.yaml

```diff
@@ -1,2 +1,77 @@
 # /nodes/{node_id}/files/
 # https://api.osf.io/v2/nodes/ezcuj/files/
+get:
+  summary: List all storage providers
+  description: >-
+    List of all storage providers that are configured for this node
+
+
+    Users of the OSF may access their data on a [number of cloud-storage services](https://api.osf.io/v2/#storage-providers) that have integrations with the OSF.
+    We call these **providers**. By default, every node has access to the OSF-provided storage but may use as many of
+    the supported providers as desired.
+
+
+
+    ####Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+
+    The `data` key contains an array of files.
+    Each resource in the array is a separate file object.
+
+
+    The `links` key contains a dictionary of links that can be used for [pagination](#Introduction_pagination).
+
+
+    Note: In the OSF filesystem model, providers are treated as folders, but with special properties that distinguish
+    them from regular folders. Every provider folder is considered a root folder, and may not be deleted through the regular file API.
+
+  parameters:
+
+    - in: path
+      name: node_id
+      type: string
+      required: true
+      description: 'The unique identifier of the node.'
+
+  tags:
+    - Nodes
+  operationId: nodes_providers_list
+  x-response-schema: File
+  responses:
+    '200':
+      description: 'OK'
+      schema:
+        type: array
+        items:
+          $ref: '../files/definition.yaml'
```

Comment:
I don't *think* the files definition is the appropriate response here since this endpoint returns a list of enabled addons? I could be wrong though because I don't even remember what's in the file definition.